src/components/transactions/HexEncodedData/index.tsx

```diff
@@ -18,22 +19,29 @@ export const HexEncodedData = ({ hexData, title, limit = 20 }: Props): ReactElem
     setShowTxData((val) => !val)
   }
 
+  const firstBytes = <b>{hexData.slice(0, 8)}</b>
+  const restBytes = hexData.slice(8)
```

Comment:
It is the first 4 bytes. So including the 0x-prefix it is 10 chars.
```suggestion
  const firstBytes = <b>{hexData.slice(0, 10)}</b>
  const restBytes = hexData.slice(10)
```